### ISSUE TITLE
It would be good if there are milestones for each release.

### ISSUE THREAD

**Issue by portlek** [Carleslc/Simple-YAML#23]

**Comment by portlek** [comment]
It would be good to know what the next release includes are. So the issues and pull requests can shape for the milestone.

**Comment by Carleslc** [comment]
Nothing relevant in 1.7 for the moment, only #15 (maybe version tag should be 1.6.2 instead?).

Added here: https://github.com/Carleslc/Simple-YAML/projects/1

**Comment by portlek** [comment]
Yes, `1.6.2` is good.

**Comment by Carleslc** [comment]
Well, other issues currently closed will be available too, like #16.

Features not released yet can be fetched using the `master-SNAPSHOT` JitPack tag.

```xml
<dependency>
    <groupId>com.github.Carleslc.Simple-YAML</groupId>
    <artifactId>Simple-Yaml</artifactId>
    <version>master-SNAPSHOT</version>
</dependency>
```

```xml
<dependency>
    <groupId>com.github.Carleslc.Simple-YAML</groupId>
    <artifactId>Simple-Configuration</artifactId>
    <version>master-SNAPSHOT</version>
</dependency>
```

Or with a commit tag:

```xml
<dependency>
    <groupId>com.github.Carleslc.Simple-YAML</groupId>
    <artifactId>Simple-Configuration</artifactId>
    <version>44ea4747b8</version>
</dependency>
```
